client/src/Videos.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+function Video(props) {
```

Comment:
If you prefer, you can destructure the props as parameters to the component, to avoid having to write `props.videoObj` within the component. i.e.
```
function Video({ videoObj, removeVideo }) {
```